### ISSUE TITLE
Didn't reproduce the results in the paper. 

### ISSUE THREAD

**Issue by hailong23-jin** [Aldrich2y/MIANet#3]
Hi, Thanks for the great work, but when I reproduce the results on PASCAL dataset, I didn't obtain the similar results in the paper.

I used the original code, training on PASCAL dataset, split 0, based on ResNet50.  This is the results:

| Paper| Ours|
| -- | --|
| 68.51 | 66.88|

Is there any other settings I need to modify in config file?

**Comment by Aldrich2y** [comment]
Hello，thank you for your interest in our work. Here are some possible suggestions to improve the model performance ：1. Ensure that the dataset settings are consistent with those in Readme. 2. Ensure that your backbone network is pretrained with PSPNet. 3. the setting "ori_resize" in the file "pascal_split0_resnet50.yaml " is "False" in testing. 4. Choose an appropriate test seed.

**Comment by hailong23-jin** [comment]
Thanks for your quick reply! Can you provide the original settings from the paper for a fair comparison. It would even be better to provide the checkpoints of the model.

**Comment by X-Hang** [comment]
I retrained with the original code, but did not get similar results to the paper. The result of my re-training is as follows:
| split0             | split0             | split1             | split2              | split3             | mean             |
| :-------------:  | :-------------:  | :-------------:   | :-------------:    | :-------------:   | :-------------:   |
| Paper            |       68.51      |     75.76       | 67.46  | 63.15  | 68.72  |
| Ours             | 66.47  | 73.71  | 64.13  | 60.65  | 66.24  |

**Comment by Aldrich2y** [comment]
Hello, here are some possible suggestions to improve the model performance ：1. Ensure that the dataset settings are consistent with those in Readme. 2. Ensure that your backbone network is pretrained with PSPNet. 3. the setting "ori_resize" in the file "pascal_split0_resnet50.yaml " is "False" in testing. 4. Choose an appropriate test seed.      

Additionally, we provided some checkpoint results of the MIANet in https://drive.google.com/file/d/1mAdNcSN8xGSmiJ0eWT2UrZl-2lHR8spM/view?usp=drive_link

**Comment by King-king424** [comment]
> Hello, here are some possible suggestions to improve the model performance ：1. Ensure that the dataset settings are consistent with those in Readme. 2. Ensure that your backbone network is pretrained with PSPNet. 3. the setting "ori_resize" in the file "pascal_split0_resnet50.yaml " is "False" in testing. 4. Choose an appropriate test seed.
> 
> Additionally, we provided some checkpoint results of the MIANet in https://drive.google.com/file/d/1mAdNcSN8xGSmiJ0eWT2UrZl-2lHR8spM/view?usp=drive_link

I used your provided weights and config to test mianet.
I got the performance as below:
spilit0: 68       (68.51 reported in paper)
split1: 74.8     (75.76 reported in paper)
This does not appear to be consistent with the precision reported in the paper

**Comment by Aldrich2y** [comment]
The computer  environment may affect the results of the experiment. It is recommended that you take different seeds for multiple experiments.

**Comment by zxmyyds** [comment]
> Hi, Thanks for the great work, but when I reproduce the results on PASCAL dataset, I didn't obtain the similar results in the paper.
> 
> I used the original code, training on PASCAL dataset, split 0, based on ResNet50. This is the results:
> 
> Paper	Ours
> 68.51	66.88
> Is there any other settings I need to modify in config
Hello I would like to ask a question Does the dataset VOC2012+SBD mean that both datasets should be put in , but I see that voc2012 is not included?

**Comment by Aldrich2y** [comment]
> > Hi, Thanks for the great work, but when I reproduce the results on PASCAL dataset, I didn't obtain the similar results in the paper.
> > I used the original code, training on PASCAL dataset, split 0, based on ResNet50. This is the results:
> > Paper	Ours
> > 68.51	66.88
> > Is there any other settings I need to modify in config
> > Hello I would like to ask a question Does the dataset VOC2012+SBD mean that both datasets should be put in , but I see that voc2012 is not included?

hello,  voc2012 has more than 10,000 images but does not have so many labels, so it is supplemented by labels from the SBD dataset.
